uint8 motor
bool enable
